core/src/main/java/com/linecorp/armeria/server/Server.java

```diff
@@ -116,6 +117,10 @@
         config.serviceConfigs().forEach(cfg -> ServiceCallbackInvoker.invokeServiceAdded(cfg, cfg.service()));
     }
 
+    public static ServerBuilder builder() {
```

Comment:
A factory method like this must appear before the member fields:

```java
public class X {

    public static XBuilder builder() { ... }

    private Foo foo;
    private Bar bar;
}
```

Please fix all other similar problems in this PR as well.